pkg/controller/migmigration/restore.go

```diff
@@ -78,15 +124,27 @@ func (t Task) getRestore() (*velero.Restore, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Find proper restore to return
 	if len(list.Items) > 0 {
-		return &list.Items[0], nil
+		for i, restore := range list.Items {
+			// Avoid nil dereference
+			if restore.Annotations == nil {
+				restore.Annotations = make(map[string]string)
+			}
+			if restore.Annotations[copyBackupRestoreAnnotationKey] != "" && copyRestore {
```

Comment:
What is the circumstance that the annotation would be in the map with a value of `""` ?  How/when/where does it get set that way.  

Or:

Is this a _Go_ thing?
The same as `_, found := restore.Annotations[copyBackupRestoreAnnotationKey]` and checking `if !found`?  If so, is it safe?

Reply:
`copyBackupRestoreAnnotationKey` is set to `""` if this is the final restore. I.e. restore #2 of backup #1. 

Yes your snippet is identical, I can change it if that is easier to read. (Probably is)